prow/prowjobs/linux-review.googlesource.com/linux-lkml-bridge-config.yaml

```diff
@@ -0,0 +1,20 @@
+presubmits:
+  https://linux-review.googlesource.com/linux:
```

Comment:
I think this line should be: `https://linux-review.googlesource.com/linux/kernel/git/torvalds/linux`

Since the name of this repo is `linux/kernel/git/torvalds/linux`: https://linux-review.googlesource.com/q/project:linux%252Fkernel%252Fgit%252Ftorvalds%252Flinux+status:open